lblod/app-lokaal-mandatenbeheer
# fix persons having duplicate birth dates

## Description

fix persons having duplicate birth dates
## How to test

run the queries linked in the ttl files and verify that they give no results after running these migrations on prod data
